website/docs/sql-reference/clauses/sql-order-by.md

```diff
@@ -0,0 +1,65 @@
+---
+id: order-by
+title: SQL ORDER BY
+description: The ORDER BY clause allows you to specify the resulting row order for a query.
+slug: /sql-reference/order-by
+---
+
+<head>
+    <title>Working with the SQL ORDER BY clause</title>
+</head>
+
+The ORDER BY clause allows you to specify the resulting row order for a query. In practice, you use the ORDER BY clause to indicate which field(s) you want to order by and in what type of order you want (ascending or descending). It’s useful to leverage during ad hoc analyses and for creating appropriate column values for partitioned rows in window functions.
+
+## How to use the SQL ORDER BY clause
+
+ORDER BY clauses have multiple use cases in analytics work, but we see it most commonly utilized to:
+- Order a query or subquery result by a column or group of columns
+- Appropriately order a subset of rows in a window function
+
+To use the ORDER BY clause to a query or model, use the following syntax:
+
+```sql
+select
+	column_1,
+	column_2
+from source_table
+order by <field(s)> <asc/desc> --comes after FROM, WHERE, and GROUP BY statements
+```
+You can order a query result by multiple columns, represented by their column name or by their column number in the select statement (ex. `order by column_2 == order by 2`). You can additionally specify the ordering type you want (ascending or descending) to return the desired row order.
```

Comment:
We should note which warehouses support column number ordering (or not note it if it's all of them)

Reply:
I believe it's all of them; tbh I think this is such a SQL fundamental we wouldn't need to call this out?